pkg/internal/checkup/vmi/vmi.go

```diff
@@ -158,6 +158,7 @@ func WithHugePages() Option {
 	return func(vmi *kvcorev1.VirtualMachineInstance) {
 		vmi.Spec.Domain.Memory = &kvcorev1.Memory{
 			Hugepages: &kvcorev1.Hugepages{PageSize: "1Gi"},
+			Guest:     &resource.Quantity{Format: "4Gi"},
```

Comment:
Could you please add a memory size parameter to `WithHugePages()`?
I think it will be nicer to see the memory size from the function that builds the VMI spec.

Reply:
I agree

Reply:
Please consider also renaming it to `WithMemory()` or something alone those lines.